cargo-miri/bin.rs

```diff
@@ -628,6 +647,8 @@ fn phase_cargo_miri(mut args: env::Args) {
     // Set rustdoc to us as well, so we can run doctests.
     cmd.env("RUSTDOC", &cargo_miri_path);
 
+    cmd.env("MIRI_LOCAL_CRATES", local_crates(&metadata));
```

Comment:
Please also document the env var in the README, where the other private env vars are documented.

Reply:
I gave this a shot, but I'm trying to be somewhat defensive because I want to use the set of local crates for other diagnostics apart from just backtraces.

Reply:
LGTM!